sorc/degrib2.fd/CMakeLists.txt

```diff
@@ -1,34 +1,22 @@
-
-if(IntelComp)
-  set(fortran_flags "-g" "-O3" "-convert" "big_endian" "-axCORE-AVX2")
-elseif(GNUComp)
-  set(fortran_flags "-g" "-O3" "-fconvert=big-endian")
-else()
-  message("unknown compiler!")
-  exit()
+if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
+  set(CMAKE_Fortran_FLAGS "-g -convert big_endian -axCORE-AVX2 -fpp")
```

Comment:
axCORE-AVX2 is useless on some machines with Intel. But OK!

Reply:
You are correct.  We still need it on WCOSS.